_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Strimzi 0.5.0 released!"
+date: 2018-07-25
+author: paolo_patierno
+---
+
+We are really happy to announce the new Strimzi 0.5.0 release with a lot of new awesome features!
+
+<!--more-->
+
+# Kafka as a native Kubernetes/OpenShift resource
+
+The first really cool feature is about the Cluster Operator moving from ConfigMaps to Custom Resources.
+It means that the Kafka and the Kafka Connect clusters aren't described by ConfigMaps anymore but through Custom Resources.
+Here's an example of the `Kafka` resource for deploying a Kafka cluster:
```

Comment:
It's not a full example (it has the `...` so isn't valid YAML). Either make it valie YAML or call it an "example snippet" (or something to highlight that you can't just `kubectl create` this thing).